styleguide/derek/examples/subpanels.ejs

```diff
@@ -0,0 +1,158 @@
+<%
+
+// List of available layouts.
+var layouts = {
+  '50_50': {
+    'title': '50/50 Layout',
+    'class_left': 'col-md-6',
+    'class_right': 'col-md-6'
+  },
+  '67_33': {
+    'title': '67/33 Layout',
+    'class_left': 'col-md-8',
+    'class_right': 'col-md-4'
+  }
+};
+
+// List of available patterns sorted by title.
+var patterns = {
+  'announcements': {'title': 'Announcements'},
+  'additional-resources': {'title': 'Additional Resources'},
+  'business-resources': {'title': 'Business Resources'},
+  'cta': {'title': 'CTA'},
+  'features': {'title': 'Features'},
+  'services-3-col': {'title': 'IAS: 3 cols'},
+  'services-4-col': {'title': 'IAS: 4 cols'},
+  'services-5-col': {'title': 'IAS: 5 cols'},
+  'services-6-col': {'title': 'IAS: 6 cols'},
+  'key-offers': {'title': 'Key Offers'},
+  'lead': {'title': 'Lead'},
+  'list': {'title': 'List'},
+  'testimonials': {'title': 'Quote'},
+  'text': {'title': 'Text'},
+  'value-prop': {'title': 'Value Prop'}
+};
+%>
```

Comment:
The way this page works is that it defines the layouts/patterns as simple JS objects. We then build out the HTML based on this info.